src/App.js

```diff
@@ -1,23 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+import ArticlesList from "./components/ArticlesList";
+import { Route, Routes } from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <Nav />
+      <main className="articles-list">
+        <ArticlesList />
+        {/* <Routes>
```

Comment:
We don't need this commented-out code until the next ticket, so it shouldn't be included in this PR. Just make sure to look over your code and remove anything that isn't needed at the moment